lib/winring0.cpp

```diff
@@ -10,9 +10,22 @@ extern "C" {
 #include "OlsDef.h"
 
 #pragma comment(lib, "../prebuilt/WinRing0x64.lib")
+#pragma comment(lib, "../prebuilt/inpoutx64.lib")
```

Comment:
Probably it's time to rename this file as osdep_win32 or sth.

Reply:
BTW you found the one line where I was unsure if we should keep this after failing with linker integration and using loading on runtime.

Should I delete this line after renaming?

Reply:
maybe

Reply:
I did remove it